workflows/dragen-pon-qc/3.9.3/dragen-pon-qc__3.9.3.cwl

```diff
@@ -0,0 +1,912 @@
+    type: File?
+    secondaryFiles:
+      - pattern: ".fai"
+        required: true
+  split:
+    label: split
+    doc: |
+      In addition to the complete statistics, also output categorised statistics based on the tagged field TAG 
+      (e.g., use --split RG to split into read groups).
+    type: string?
+  target_regions:
+    label: target regions
+    doc: |
+      Do stats in these regions only. Tab-delimited file chr,from,to, 1-based, inclusive. []
+    type: File?
+  sparse:
+    label: sparse
+    doc: |
+      Suppress outputting IS rows where there are no insertions.
+    type: boolean?
+  remove_overlaps:
+    label: remove overlaps
+    doc: |
+      Remove overlaps of paired-end reads from coverage and base count computations.
+    type: boolean?
+  cov_threshold:
+    label: cov threshold
+    doc: |
+      Only bases with coverage above this value will be included in the target percentage computation [0]
+    type: int?
+  threads:
+    label: threads
+    doc: |
+      Number of input/output compression threads to use in addition to main thread [0].
+    type: int?
+  # PRECISE QC step
+  script:
+    label: script
+    doc: |
+      Path to PRECISE python script on GitHub
+    type: File
+    default:
+      class: File
+      location: https://raw.githubusercontent.com/c-BIG/wgs-sample-qc/main/example_implementations/sg-npm/calculate_coverage.py
+  map_quality:
+    label: map quality
+    doc: |
+      Mapping quality threshold. Default: 20.
+    type: int?
+  out_directory:
+    label: out directory
+    doc: |
+      Path to scratch directory. Default: ./
+    type: Directory?
+  output_json:
+    label: output json
+    doc: |
+      output file
+    type: string?
+  log_level:
+    label: log level
+    doc: |
+      Set logging level to INFO (default), WARNING or DEBUG.
+    type: string?
+  # custom-qc step that combines Singapore's and UMCCR's QC outputs
+  sample_id:
+    label: sample id
+    doc: |
+      Sample identity
+    type: string
+  sample_source:
+    label: sample source
+    doc: |
+      Sample original source
+    type: string
+  output_json_filename:
+    label: output filename
+    doc: |
+      output file
+    type: string?
+
+steps:
+  # Run dragen somatic tool using normal fastqs
+  run_dragen_somatic_step:
+    label: run dragen somatic step
+    doc: |
+      Runs the dragen somatic workflow on the FPGA.
+      Takes in a normal and tumor fastq list and corresponding mount paths from the predefined_mount_paths.
+      All other options avaiable at the top of the workflow
+    in:
+      tumor_fastq_list:
+        source: tumor_fastq_list
+      tumor_fastq_list_rows:
+        source: tumor_fastq_list_rows
+      reference_tar:
+        source: reference_tar
+      output_directory:
+        source: output_directory
+      output_file_prefix:
+        source: output_file_prefix
+      enable_map_align_output:
+        source: enable_map_align_output
```

Comment:
As discussed in meeting, change from `source: enable_map_align_output` to 

```
valueFrom: |
  ${ return true; }
```

Reply:
fixed in https://github.com/umccr/cwl-ica/pull/85/commits/f501c4effabbf62bb55084de6998dd12789dc882